source/hyperparameter_optimization/config/SplitConfig.py

```diff
@@ -27,7 +27,7 @@ class SplitConfig:
 
         training_percentage: if split_strategy is RANDOM, this defines which portion of the original dataset to use for creating the training dataset; for other values of split_strategy, this parameter is not used.
 
-        reports (ReportConfig): defines which reports to execute on which datasets or settings. See ReportConfig for more details.
+        reports (ReportConfig): defines which reports to execute on which datasets or settings. See :py:obj:`~source.hyperparameter_optimization.config.ReportConfig.ReportConfig` for more details.
```

Comment:
i see that you add a lot of these references. in theory that's good, but it will redirect the user to the object docs (under developer docs) instead of a bit further down the specification page with more user-friendly docs with YAML (which is what user would typically want).

Reply:
ok, but in that case should the whole message of 'see ReportConfig for details' just be removed? The only difference now is that we link to ReportConfig, but what you are saying is that they don't want to look at ReportConfig for details

Reply:
no, it's just the matter of where we want to redirect them: to dev docs (where this would redirect now) or to the specs page but different heading. just checked: there is a bug in the docs so this is not displayed (i will fix it now), but i think that's where it should lead, if you agree? :)

Reply:
updated now!